pages/advanced_commands.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Advanced commands
+---
+
+## Advanced commands start with `<` and when selected expand a BEGIN...END section inside a block. There can be multiple BEGIN...END sections in a block.
+## List of advanced commands:
+### `<Quote` - Displays text as quoted by indenting text and giving it left and right margins.
+### `<Src` - Syntax highlights code text using CodeMirror. Passing an argument to the block e.g. `#+BEGIN_SRC clojure` sets the language. Default language is javascript.
+### `<Query` - Defines a block for [[Queries]]. Block renders query results when not being edited.
+### `<Latex export` - TODO
```

Comment:
I left TODOs for the ones I was unsure about

Reply:
Cool, the latex export will be something like:
```
#+BEGIN_EXPORT latex
E = mc2 
#+END_EXPORT
```
It'll be rendered by using Katex.